packages/devextreme-vue/src/core/index.ts

```diff
@@ -1,11 +1,7 @@
-import * as vue2Stategy from './strategy/vue2/index';
-import * as vue3Stategy from './strategy/vue3/index';
-import { isVue3 } from './version';
+import * as vue3Strategy from './strategy/vue3/index';
```

Comment:
Do we need strategy folder? Looks like we can move 'prepareComponentConfig', 'prepareConfigurationComponentConfig' and 'prepareExtensionComponentConfig' to this module

Reply:
yes it can be removed